site/_wiki/Install/MacOS.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: macOS Installation Guide
+title: Intel macOS Installation Guide
```

Comment:
Probably add a subheading to App instead of modifying the title to keep it consistent with other install guide titles.

Reply:
How about "macOS (Intel) Installation Guide" for `title` instead?

Reply:
Yep, that works too.